keel-actuator/src/main/kotlin/com/netflix/spinnaker/keel/actuation/ResourceActuator.kt

```diff
@@ -55,6 +63,11 @@ class ResourceActuator(
 
     log.debug("Checking resource {}", id)
 
+    if (resourceRepository.lastEvent(id) is ResourceActuationPaused) {
```

Comment:
Should we care about `ResourceActuationResumed`? I am leaning no - here's my logic.

We're only updating this status every time we check the resources, so if we emit a `ResourceActuationResumed` event here we will immediately below emit another event with the state of the resource. Because of that, `ResourceActuationResumed` will never be the last status for more than a couple of seconds. Additionally, we don't get any status information from `ResourceActuationResumed` - we don't know if the resources is happy, has a diff, etc. 

Is there any case I'm not thinking of where it's helpful to know that a resource was paused but has just switched?

Reply:
My thinking was that it might be useful to surface that event on the UI so you have a complete picture of when actuation was paused and then resumed. You’re right that we emit other events immediately after, but it might not be immediately obvious that they happened as a result on removing the veto/resuming.

Having said that, I’m only becoming familiar with the event history and how it’s used, so if it’s overkill I’d be happy to remove it.

Reply:
@erikmunson Do we consume the event history directly from the UI today, or only resource status?

Reply:
As of this moment we’re only linking out to the history endpoint for a resource as raw JSON, but it is available to people and the next thing I’ll be working on is consuming it for a UI view.

Either way (JSON or UI), 100% agreed that having a resume event in the activity feed itself has value to human readers that are trying to understand the narrative timeline of what happened. I don’t see a reason to think about it when the target audience is just code/machines, but it should be present in the history we expose publicly.

Reply:
Fair points! Let's leave it in then.